scripts/install

```diff
@@ -116,49 +112,49 @@ ynh_add_fpm_config
 #=================================================
 # CONFIGURE WP-CONFIG
 #=================================================
-ynh_script_progression --message="Configure wp-config.php"
+ynh_script_progression --message="Configuring wordpress..."
 
 cp ../conf/wp-config.php $final_path/wp-config.php
 # Change variables in Wordpress configuration
-ynh_replace_string "__DB_USER__" "$db_name" $final_path/wp-config.php
-ynh_replace_string "__DB_PWD__" "$db_pwd" $final_path/wp-config.php
+ynh_replace_string --match_string="__DB_USER__" --replace_string=$db_name --target_file=$final_path/wp-config.php
+ynh_replace_string --match_string="__DB_PWD__" --replace_string="$db_pwd" --target_file=$final_path/wp-config.php
 
 for i in 1 2 3 4 5 6 7 8
 do
-	j=$(dd if=/dev/urandom bs=1 count=200 2> /dev/null | tr -c -d '[A-Za-z0-9]' | sed -n 's/\(.\{40\}\).*/\1/p')
+	j=$(ynh_string_random --length=40)
 	if [ "$j" = "" ];
```

Comment:
Maybe this **if** is no longer necessary

Reply:
That's a very old part of the code, should be tried.

Reply:
indeed

Reply:
[![Build Status](https://ci-apps-dev.yunohost.org/jenkins/job/wordpress_ynh%20PR64/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/wordpress_ynh%20PR64/)

Reply:
```
Info:  [######..............] > Configuring wordpress...
Info:  for loop: SpJzljnlJhEMbRV5Qg4kDUmjNBbh6B7DiXamLPjs
Info:  for loop: dPqaPZoIZY4hpjBHxdGMYWO7Z9MKIZB6QOTYBZh7
Info:  for loop: Dq09CUXixCdRc1cqjEsEgnywhmjjPk1lj77BIZu1
Info:  for loop: SPGavq4ARYbY11zrrrwoKbLDZ2XplJKkotCW8jgT
Info:  for loop: qmrdAT4UV8lButEPO4fs9m86su0CSfsO4wLeCYVW
Info:  for loop: GPmqdxoZf2GSRshGtGao2aCTWV8OE7B9DruW0aox
Info:  for loop: riWusw0uF89firJTU8sL6Z9XL41aCrWkWVZx5Hxz
Info:  for loop: ZegzgwMsfr4E5Z5f1A6pv8CUvzyrSuJpPMq8pIVj
Info:  [######..............] > Installing wordpress with Curl...
```

For now it's ok... But maybe it's hard to reproduce this bug :-/

Reply:
I guess a simple `sleep 0.5` between each loop would be enough to be sure.